docker-keys.yml

```diff
@@ -0,0 +1,9 @@
+secrets:
```

Comment:
Also these files are the wrong way around.
`docker-keys.yml` should contain keys not secrets and visa versa